package.json

```diff
@@ -1254,7 +1254,8 @@
     "p-retry": "^4.2.0",
     "papaparse": "^5.5.3",
     "pbf": "3.2.1",
-    "pdfmake": "^0.2.15",
+    "pdf-lib": "^1.17.1",
```

Comment:
pretty sure `pdf-lib` shouldn't be in here; it leaked into one of the other PRs as well, and I removed it from that one, thought I had gotten this one too.  In a minute ... :-)